api/src/main/java/com/okta/sdk/ds/DataStore.java

```diff
@@ -113,4 +113,11 @@
      */
     RequestBuilder http();
 
+    /**
+     * Check if Datastore is configured correctly and able to execute requests.
+     *
+     * @return {@code true} if orgUrl and token are correct and Okta server is available, otherwise {@code false}.
```

Comment:
Nit: expand a bit on `Okta server is available`.

Reply:
Here I meant not a particular server but Okta Core in general and _available_ means "able to be used or obtained". I think everything is quite clear here. Please, feel free to post your variant of the description.